cetmix_tower_server/wizards/cx_tower_command_execute_wizard_view.xml

```diff
@@ -97,7 +101,7 @@
                         help="Run code as it appears in 'Rendered code' in wizard and return to wizard. Result will not be logged"
                         class="oe_highlight"
                         groups="cetmix_tower_server.group_manager"
-                        attrs="{'invisible': ['|', ('result', '!=', False),('code', '=', False)]}"
+                        attrs="{'invisible': [('show_servers', '=', True), '|', '|', ('result', '!=', False), ('code', '=', False), ('command_id', '!=', False)]}"
```

Comment:
This domain means the following:

```
show_servers=True
AND
(
    result!=False OR code=False OR command_id!=False
)
```

Which result in the following behaviour:
![image](https://github.com/user-attachments/assets/4a6fe64e-792f-43d1-b444-dc9d0b3bdcea)